spec/jasmine/ts.css

```diff
@@ -3075,7 +3075,7 @@ i.ts-icon svg {
 .ts-form label.ts-disabled .ts-field.ts-text {
   border-color: #e1e1e1;
 }
-.ts-form label.ts-customiconlabel:not(.ts-option):after {
+.ts-form label.ts-customiconlabel:not(.ts-option):not(.ts-switchlabel):after {
```

Comment:
This stuff should have already been merged to master :north_korea: Rebase rulez!